docs/content/tutorials/ingress-path-regex-annotation.md

```diff
@@ -0,0 +1,457 @@
+---
+title: Customizing Ingresses with Path-Regex Annotations
+description: |
+  How to customize Simple Ingress and Mergeable Ingress types with path-regex annotations.
+weight: 1800
+doctypes: ["concept"]
+toc: true
+---
+## Customizing NGINX Ingress Controller with Path-Regex Annotations
+
+This document explains how to use `path-regex` annotations with Simpe Ingress and Mergeable Ingress (Master and Minions)
+types.
+
+[NGINX documentation](https://docs.nginx.com/nginx/admin-guide/web-server/web-server/#nginx-location-priority) provides
+additional information about how NGINX and NGINX Plus resolve location priority.
+Read [it](https://docs.nginx.com/nginx/admin-guide/web-server/web-server/#nginx-location-priority) before using
+ the ``path-regex`` annotation.
+
+## Simple Ingress Type
+
+In this example, you will use the `nginx.org/path-regex` annotations to add regex modifiers the location paths.
+
+Start by modifying `cafe-ingress.tmpl` metadata. Add the annotation section and configure annotation ``nginx.org/path-regex``
+with value `case_sensitive`.
+
+`cafe-ingress.tmpl`:
+
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: cafe-ingress
+  annotations:
+    nginx.org/path-regex: "case_sensitive"
+spec:
+  tls:
+  - hosts:
+    - cafe.example.com
+    secretName: cafe-secret
+  rules:
+  - host: cafe.example.com
+    http:
+      paths:
+      - path: /tea/[A-Z0-9]
+        backend:
+          serviceName: tea-svc
+          servicePort: 80
+      - path: /coffee/[A-Z0-9]
+        backend:
+          serviceName: coffee-svc
+          servicePort: 80
+```
+
+After creating the Ingress (`kubectl create -f cafe-ingres,yaml`), all defined paths will be updated. In the generated
+NGINX config file the ``tea`` and ``coffee`` paths will look like in the snippets below:
+
+tea path:
+
+```bash
+location ~ "^/tea/[A-Z0-9]" {
+```
```

Comment:
I would change the code type from `bash` to `nginx` for both of these to keep it consistent with the other examples.
I would also remove the curly brace at the end of the line.

```suggestion
tea path:

```nginx
location ~ "^/tea/[A-Z0-9]"
```
```